### PR TITLE
Bump up versions of yui-relayer (v0.4.7), ethereum-ibc-relay-chain (v0.4.3) , and ethereum-ics20-cli (v0.0.4)

### PR DESCRIPTION
**Pull request by dongrie**

### CODE REVIEW

File: tests/cases/eth2eth/scripts/util/relayer-util [hunk 1]
```diff
@@ -18,3 +18,21 @@ CHAINID_TWO=ibc1
...
+  do
+      echo "[try:$i] waiting for ${label} finalization ..."
+      sleep $RETRY_INTERVAL
+      unrelayed=$(${RLY} query ${query_type} ${PATH_NAME} | jq "${filter}")
+      if [ $unrelayed -gt 0 ]
+      then
+    break
+      fi
+  done
+}
```
Comment (siburu) [inline tests/cases/eth2eth/scripts/util/relayer-util:38]: Thank you for adding this one.
I like it, but indents are not aligned?

File: tests/chains/ethereum/contract/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "clean": "rimraf build"
   },
   "dependencies": {
-    "@hyperledger-labs/yui-ibc-solidity": "git+https://github.com/hyperledger-labs/yui-ibc-solidity.git#v0.3.8",
+    "@hyperledger-labs/yui-ibc-solidity": "git+https://github.com/hyperledger-labs/yui-ibc-solidity.git#v0.3.10",
```
[the remark this answers is not in the document]
Comment (dongrie) [reply tests/chains/ethereum/contract/package.json:14]: I checked the diffs for v0.3.8 and v0.3.10. No migration related changes were found.
Screenshot 2023-09-08 at 10 55 51

[the remark this answers is not in the document]
Comment (siburu) [reply tests/chains/ethereum/contract/package.json:14]: Great. Thank you.

File: tests/chains/ethereum/Dockerfiles/ganache/run.sh [hunk 3]
```diff
@@ -6,4 +6,5 @@ node "/app/ganache-core.docker.cli.js" \
   --db /root/.ethereum \
   --defaultBalanceEther 10000 \
   --mnemonic "math razor capable expose worth grape metal sunset metal sudden usage scheme" \
+  --blockTime 1 \
```
Comment (siburu) [inline tests/chains/ethereum/Dockerfiles/ganache/run.sh:9]: I see. Ganache doesn't generate new blocks without transactions.

**Review by siburu — APPROVED** [review]
LGTM!

**Pull request merged**